ci-operator/config/redhat-openshift-ecosystem/community-operators-prod/redhat-openshift-ecosystem-community-operators-prod-main__4.6.yaml

```diff
@@ -0,0 +1,45 @@
+build_root:
+  project_image:
+    dockerfile_path: scripts/ci/Dockerfile.ci-operator
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.6"
+resources:
+  '*':
+    requests:
+      cpu: 1000m
+      memory: 4Gi
+tests:
+- as: deploy-operator-on-openshift
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws
+    test:
+    - as: deploy-operator
+      commands: |
+        export OCP_CLUSTER_VERSION=4.6
+        mkdir -p "${HOME}"
+        scripts/ci/openshift-deploy.sh
+      credentials:
+      - mount_path: /var/run/cred
+        name: tst
```

Comment:
It would be really really nice if you took this PR as an opportunity to change this very poor name for a secret. When you log in to https://vault.ci.openshift.org/ui/ and find the `pipeline-secrets` collection (you should be able to access it, Vault was populated from the previous secret mirroring configuration), there's a `tst` secret there, and it has a `secretsync/target-name` key that's currenty set to `tst`. It would be really great if you changed that to something more indicative of what's the secret's purpose.

Reply:
fixed in this PR, Vault will be fixed after merge to minimize downtime.